cmd/nitro/nitro.go

```diff
@@ -719,6 +723,9 @@ func mainImpl() int {
 		return 1
 	}
 
+	if quitNodeMsg != "" {
+		log.Info("quitting node", "msg", quitNodeMsg)
```

Comment:
nitpick: we use "shutdown node" instead of "quitting node" in other places, it would be nice to have a standard on that

Reply:
Done.